sbi/inference/trainers/vfpe/simformer.py

```
@@ -0,0 +1,192 @@
+            device=device,
+            logging_level=logging_level,
+            summary_writer=summary_writer,
+            show_progress_bars=show_progress_bars,
+            sde_type=sde_type,
+            **kwargs,
+        )
+
+    def _build_default_nn_fn(self, **kwargs) -> MaskedVectorFieldEstimatorBuilder:
+        net_type = kwargs.pop("vector_field_estimator_builder", "simformer")
+        return simformer_nn(model=net_type, **kwargs)
+
+    def build_arbitrary_joint(
+        self,
+        conditional_mask: Tensor,
+        edge_mask: Tensor,
+        masked_vector_field_estimator: Optional[
+            MaskedConditionalVectorFieldEstimator
+        ] = None,
+        prior: Optional[Distribution] = None,
+        sample_with: str = "sde",
+        **kwargs,
+    ) -> VectorFieldJoint:
+        r"""Build an arbitrary conditional joint distribution from
+        the vector field estimator.
+
+        Args:
+            masked_vector_field_estimator: The vector field estimator that the posterior
+                is based on. If `None`, use the latest vector field estimator that was
+                trained.
+            prior: Prior distribution (unused).
+            sample_with: Method to use for sampling from the posterior. Can only be
+                'sde' (default).
+            **kwargs: Additional keyword arguments passed to
+                `VectorFieldBasedPotential`.
+
+        Returns:
+            Conditional distribution of latent nodes given the observed nodes
+            and the edge structure.  With `.sample()` and `.log_prob()` methods.
+        """
+        return self._build_arbitrary_joint(
+            conditional_mask,
+            edge_mask,
+            masked_vector_field_estimator,
+            prior,
+            sample_with=sample_with,
+            **kwargs,
+        )
+
+    def build_posterior(
+        self,
+        condition_mask: Tensor,
+        edge_mask: Tensor,
```

Can we make `edge_mask: Optional[Tensor]=None`, and it defaults to all ones. 

Actually, there is also something we have to keep in mind:
- If we train on, e.g., a specific undirected edge mask. Then we should also default to this mask! (because using something else at inference time can lead to worse performance).
- This would require maniging a "default_mask", but I think for now its fine to leave managing this to the user.